package.json

```diff
@@ -30,6 +30,9 @@
 	],
 	"author": "Tim Deschryver",
 	"license": "MIT",
+	"peerDependencies": {
+		"zod": ">=3.0.0"
+	},
 	"dependencies": {
 		"@paralleldrive/cuid2": "^2.2.0",
 		"zod": "^3.21.4"
```

Comment:
We can also move the zod dependency to devDependencies. IE, we use zod for development, but don't bundle it with the production version because it's implied that you already have it installed.